carbon-design-system/carbon-addons-iot-react
# fix(page-title-bar): fix accessibility error for `span` in `ol`

Closes #3087 

**Summary**

- moves current page into a `li` tag (instead of a `span`) and adjusts the styling to match

**Change List (commits, features, bugs, etc)**

- changes `span` into `li`
- updates tests to reflect
- updates styles to match

**Acceptance Test (how to verify the PR)**

- open the [page title bar story with dynamic scrolling](https://deploy-preview-3207--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-pagetitlebar--with-dynamic-scrolling)
- change to the accessibility tab
- click run test
- confirm no errors

**Regression Test (how to make sure this PR doesn't break old functionality)**

- check breadcrumb stories to confirm styles match next when in condensed or dynamic mode and scrolling down page



**Things to look for during review**
